proxy/api/src/http/source.rs

```diff
@@ -680,7 +682,8 @@ mod test {
         let urn = platinum_project.urn();
 
         let branch = git::Branch::local("master");
-        let head = "223aaf87d6ea62eef0014857640fd7c8dd0f80b5";
+        let head = coco::oid::Oid::try_from("223aaf87d6ea62eef0014857640fd7c8dd0f80b5")
```

Comment:
It's amazing that `coco::oid::Oid` only uses 4 distinct letters.

Reply:
By desing, lol.

Reply:
lol. It's pronounced "cocoid".